pyaedt/edb_core/general.py

```diff
@@ -50,12 +51,12 @@ def convert_pytuple_to_nettuple(_tuple):
 
 
 @pyaedt_function_handler()
-def convert_pydict_to_netdict(dict):
+def convert_pydict_to_netdict(input_dict):
     """Convert a Python dictionarty to a Net dictionary.
```

Comment:
```suggestion
    """Convert a Python dictionary to a .NET dictionary.
```
This assumes that you meant .NET, but it seems so given its a language conversion.